source/analysis/AM1DCalibrationAB.cpp

```diff
@@ -358,7 +358,7 @@ void AM1DCalibrationAB::computeCachedValues() const
 			scale = 1;
 
 		    else
-			scale = (data.at(postEdgePointValue)/normalizer.at(postEdgePointValue)) - offset;
+			scale = qAbs((data.at(postEdgePointValue)/normalizer.at(postEdgePointValue)) - offset);
```

Comment:
Thanks for drawing this to my attention!  The only reason this did solve the problem is that I had fixed the fluorescent mode algorithm and didn't fox the transmission mode one.  I did so and pushed up that change.